CONTRIBUTING.md

```diff
@@ -33,8 +33,8 @@ For more details, [see the official docs](https://developer.mozilla.org/Add-ons/
 
 * Text assets to translate: [`assets/texts`](assets/texts)
 * Screenshots: [`assets/screenshots`](assets/screenshots)
-* Wiki to translate: [wiki](/wiki)
-* Sidebar file for adding language: [`_Sidebar` file](/wiki/_Sidebar/_edit)
+* Wiki to translate: [wiki](https://github.com/rugk/awesome-emoji-picker/wiki)
```

Comment:
```suggestion
* Wiki to translate: [wiki](../wiki)
```

should be better as it is relative, is not it?

Reply:
We come from `blob/master/CONTRIBUTING.md`, so it would be `../../`, but good point